inits.json

```diff
@@ -0,0 +1,54 @@
+{
+    "user_info": [
+        {
+            "Directory with FITS files": "./sample-data/HatP32Dec202017",
+            "Directory to Save Plots": "./sample-data",
+            "Directory of Flats": "n",
+            "Directory of Darks": "n",
+            "Directory of Biases": "n",
+
+            "AAVSO Output?": "y",
+            "AAVSO Observer Account Number": "RTZ",
+            "Secondary Observer Codes": "n",
+
+            "Observation date": "December 17, 2017",
+            "Obs. Latitude (+=N,-=S)": "+31.68",
+            "Obs. Longitude (+=E,-=W)": "-110.88",
+            "Obs. Elevation (meters)": 2616,
+
+            "Camera Type (CCD or DSLR)": "CCD",
+            "Pixel Binning": "1x1",
+            "Exposure Time (seconds)": "60",
```

Comment:
So why is this value in quotations while the `Obs. Elevation` is not? They are both floats, so they shouldn't need the quotation marks.

Also...we should consider just grabbing the exposure time from the FITS header. That way we can do a quick check to make sure that the user has all of their images at the same exposure time. If not, then we can scale the images to the same exposure time, assuming n_photons ~= n_time

Reply:
Exposure times will be opened in a separate ticket as there needs to be more tests with different headers and how they spell label exposure.